Scripts/Runtime/Entities/TaskSystem/AbstractTaskDriver.cs

```diff
@@ -36,6 +38,10 @@ public abstract class AbstractTaskDriver : AbstractAnvilBase
     {
         private readonly List<AbstractTaskDriver> m_SubTaskDrivers;
         private readonly TaskFlowGraph m_TaskFlowGraph;
+        private readonly List<AbstractJobConfig> m_JobConfigs;
+        private readonly string m_TypeString;
```

Comment:
🚲 Rename to TypeName?

How often is this used? Is it actually worth caching?

Reply:
Removed